setup.py

```diff
@@ -63,6 +63,7 @@
             "load_override_data = lms.djangoapps.course_blocks.transformers.load_override_data:OverrideDataTransformer",
             "content_type_gate = openedx.features.content_type_gating.block_transformers:ContentTypeGateTransformer",
             "access_denied_message_filter = lms.djangoapps.course_blocks.transformers.access_denied_filter:AccessDeniedMessageFilterTransformer",
+            "video_url = lms.djangoapps.course_api.blocks.transformers.video_urls:VideoBlockURLTransformer",
```

Comment:
Once you move `VideoBlockURLTransformer` to be within `BlocksAPITransformer`, you can delete this line.